Monika After Story/game/script-topics.rpy

```diff
@@ -67,6 +67,52 @@ label monika_god:
     m "So, from that perspective..."
     m 5a "I don't think it's too far-fetched for there to be a god - if Earth was nothing but his playset."
     return
+    
+init 5 python:
+    #This is a list of keywords for this topic
+    for key in ['hamlet', 'shakespeare']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_hamlet') #id
+    monika_random_topics.append('monika_hamlet')
+
+label monika_hamlet:
+    m1d "[player], do you enjoy Shakespeare?"
```

Comment:
there needs to be a space between `m` and the expression.

Reply:
Whoopsie